sotoki/sotoki.py

```diff
@@ -1652,7 +1652,7 @@ def run():
         url = domain
```

Comment:
Here we can still have url assign to "http://" and not "https://".
I think the best way to process is to put the assignment to `url` outside of the if, with `url = "https://" + domain` after

Reply:
Ah I see. :thinking: That'll be a small fix. On it! :rocket:

Reply:
@dattaz just a little bit of query, use this :point_down: or use `elif`? I'm a bit confused over here :sweat_smile: 

```
    if re.match("^https?://", url):
        url = domain
        domain = re.sub("^https?://", "", domain).split("/")[0]
    else:
        url = "http://" + domain
    publisher = arguments["<publisher>"]

    if re.match("^https?://", domain):
        url = domain
        domain = re.sub("^https?://", "", domain).split("/")[0]
    else:
        url = "https://" + domain
    publisher = arguments["<publisher>"]
```

Reply:
`if re.match("^https?://", url):` check if url start with http:// or https:// to then extract the domain only.
I was thinking of :
`
    if re.match("^https?://", url):
        domain = re.sub("^https?://", "", domain).split("/")[0] 
    url = "https://" + domain
`